cs/src/YiiConfig.php

```diff
@@ -98,20 +109,20 @@ public function __construct($name = 'yii-cs-config')
             'non_printable_character' => true,
             'normalize_index_brace' => true,
             'object_operator_without_whitespace' => true,
-//            'ordered_class_elements' => [ // needs more discussion
-//                'order' => [
-//                    'use_trait',
-//                    'constant_public',
-//                    'constant_protected',
-//                    'constant_private',
-//                    'property_public',
-//                    'property_protected',
-//                    'property_private',
-//                    'construct',
-//                    'destruct',
-//                    'magic',
-//                ],
-//            ],
+            'ordered_class_elements' => [
```

Comment:
I think thing config part was under discussion because in some classes we have private properties near methods, that use them for caching etc.

Reply:
Some classes already use this rule. See [Component](https://github.com/yiisoft/yii2/blob/master/framework/base/Component.php).
It will be more consistency.

Reply:
See https://github.com/yiisoft/yii2/pull/14679

Reply:
Ok, I see. Is it possible to exclude some rules?
```
'ordered_class_elements' => [
    'order' => [
        'use_trait',
        'constant_public',
        'constant_protected',
        'constant_private',
       //'property_public',
       //'property_protected',
       //'property_private',
        'construct',
        'destruct',
        'magic',
    ],
],
```
@rob006, @SilverFire, What do you think about it?

Reply:
No, it's https://github.com/yiisoft/yii2/pull/15718#discussion_r169567238 wrong. Then I can delete this rule, because in this case the properties are behind magic methods.